File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeFilterPostcodes.scala

package org.openapitools.client.model


case class ResponseTimeFilterPostcodes (
    _results: List[ResponseTimeFilterPostcodesResult]
)
object ResponseTimeFilterPostcodes {
    def toStringBody(var_results: Object) =
        s"""
        | {
        | "results":$var_results
        | }
        """.stripMargin
}

File: fsharp-functions/OpenAPI/src/model/ResponseTimeFilterPostcodes.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.ResponseTimeFilterPostcodesResult

module ResponseTimeFilterPostcodes = 

  //#region ResponseTimeFilterPostcodes

  [<CLIMutable>]
  type ResponseTimeFilterPostcodes = {
    [<JsonProperty(PropertyName = "results")>]
    Results : ResponseTimeFilterPostcodesResult[];
  }
  
  //#endregion
  